.github/workflows/reusable-azure-ai-translate.yml

```diff
@@ -93,16 +93,45 @@ jobs:
         id: datetime
         run: echo "timestamp=$(date -u '+%B %d, %Y at %H:%M UTC')" >> "$GITHUB_OUTPUT"
 
-      - name: Create Pull Request
+      - name: Create Pull Request to update translations
         if: steps.check_pr.outputs.found == 'false'
-        uses: peter-evans/create-pull-request@271a8d0340265f705b14b6d32b9829c1cb33d45e  # v7.0.8
-        with:
-          token: ${{ secrets.NEWFOLD_ACCESS_TOKEN }}
-          commit-message: Auto-translated PO and JSON files
-          branch: ${{ steps.vars.outputs.BRANCH }}
-          title: Updating translation files on ${{ steps.datetime.outputs.timestamp }}
-          body: |
-            This pull request contains automatic translations of PO and JSON files.
-
-            The workflow run responsible for creating this pull request can be found [here](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).
-          labels: translations
+        env:
+          GH_TOKEN: ${{ secrets.NEWFOLD_ACCESS_TOKEN }}
```

Comment:
The `NEWFOLD_ACCESS_TOKEN` is not required here. If we change `persist-credentials` to `true` for the `actions/checkout` step, the regular `GITHUB_TOKEN` can be used because both `pull-requests` and `contents` are set to `write`.

The same should be true for the `check_pr` step above. This should eliminate the need for the `NEWFOLD_ACCESS_TOKEN` entirely.